assets/src/blocks/Spreadsheet/SpreadsheetFrontend.js

```diff
@@ -0,0 +1,196 @@
+import {useState, useEffect} from '@wordpress/element';
+import {ArrowIcon} from './ArrowIcon';
+import {HighlightMatches} from '../HighlightMatches';
```

Comment:
Shouldn't this be in some sort of `components` or `functions` folder?

Reply:
From the idea @GP-Dan-Tovbein brought up in slack we can move it to the `block-editor` directory

Reply:
This is used in the frontend, not in the editor, so that wouldn't make much sense